src/WebJobs.Script.WebHost/App_Data/secrets/host.json

```diff
@@ -1,4 +1,4 @@
 ﻿{
-  "masterKey": "t8laajal0a1ajkgzoqlfv5gxr4ebhqozebw4qzdy",
-  "functionKey": "zlnu496ve212kk1p84ncrtdvmtpembduqp25ajjc"
+    "masterKey": "t8laajal0a1ajkgzoqlfv5gxr4ebhqozebw4qzdy,z3dlq50s00cb3q3k11nil7xyt29ebst2n8rtn0t3",
```

Comment:
Ideally, this would be a dictionary/json object allowing people to have "named keys", as that would make the rollover and revocation process much easier (as you'd know which keys to remove)